<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pushButtonWrite</class>
 <widget class="QMainWindow" name="pushButtonWrite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>566</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(210, 75, 255);
selection-color: rgb(170, 170, 255);
gridline-color: rgb(170, 170, 255);
border-color: rgb(170, 85, 255);
background-color: rgb(29, 29, 29);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>551</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
gridline-color: rgb(170, 170, 255);
border-color: rgb(170, 170, 255);</string>
       </property>
       <property name="rowCount">
        <number>20</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Название</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Группа</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>Цена покупки</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>Цена продажи</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>160</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 127);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
